miqcli/constants.py

```diff
@@ -63,9 +63,47 @@
     'enable_ssl_verify': False
 }
 
+OS_TYPE = "ManageIQ::Providers::Openstack::CloudManager"
```

Comment:
Does OS here mean OpenStack? If so, shouldn't be better if we use only one prefix such as OPENSTACK or OSP? OS could mean lots of other things.

Reply:
I agree this will be updated to OSP.